terraform/modules/cluster/mig-with-container/variables.tf

```diff
@@ -0,0 +1,268 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+variable "project_id" {
+  description = "GCP Project ID to which the cluster will be deployed."
+  type        = string
+}
+
+variable "resource_prefix" {
+  description = "Arbitrary string with which all names of newly created resources will be prefixed."
+  type        = string
+}
+
+variable "target_size" {
```

Comment:
i was trying to follow the language of the underlying terraform modules so that users can debug a bit easier if need be. node_count is used in the other two since that is the language of those terraform resources. do you think target_size would be confusing for people?

Reply:
makes sense.